bindings/fhex/ghex_unstructured_mod.f90

```diff
@@ -0,0 +1,221 @@
+!
+! ghex-org
+!
+! Copyright (c) 2014-2023, ETH Zurich
+! All rights reserved.
+!
+! Please, refer to the LICENSE file in the root directory.
+! SPDX-License-Identifier: BSD-3-Clause
+!
+MODULE ghex_unstructured_mod
```

Comment:
Ideally we have some documentation on how to use this... I hope the test will show how to use it...